site/docs/releases.md

```diff
@@ -71,6 +71,35 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   ...
 </dependencies>
 ```
+
+### 1.6.0 release
+Apache Iceberg 1.6.0 was released on July 23, 2024.
+
+The 1.6.0 release contains fixes and dependency updates.
+
+* Build
+    - Upgrade to Gradle 8.9 ([\#10686](https://github.com/apache/iceberg/pull/10686))
+    - Move to goooler shadow plugin ([\#10568](https://github.com/apache/iceberg/pull/10568))
+* Core
+    - Fix NPE during conflict handling of NULL partitions ([\#10680](https://github.com/apache/iceberg/pull/10680))
+    - Fix ParallelIterable memory leak where queue continues to be populated even after iterator close ([\#9402](https://github.com/apache/iceberg/pull/9402))
+    - REST Catalog: deprecate oauth/tokens endpoint ([\#10603](https://github.com/apache/iceberg/pull/10603))
+    - REST Catalog: fix create v1 table ([\#10369](https://github.com/apache/iceberg/pull/10369))
+    - REST Catalog: disallow overriding "credential" in table sessions ([\#10345](https://github.com/apache/iceberg/pull/10345))
+    - REST Catalog: assume issued_token_type is access_token to fully comply with RFC 6749 )[\#10314](https://github.com/apache/iceberg/pull/10314))
+    - JDBC Catalog: exclude unexpected namespaces ([\#10498](https://github.com/apache/iceberg/pull/10498))
+* Spark
```

Comment:
given that there aren't many things listed here it gives the impression that it's a small release. I would probably just add a buch of more highlights from each component from https://github.com/apache/iceberg/releases/tag/apache-iceberg-1.6.0 (e.g. Flink is completely missing)

Reply:
+1, 
we both gave same comment at same time.

Reply:
Yeah, good point, I missed the flink ones. Let me update it.

Reply:
I updated the highlights mixing community sync notes and full changelog.